Tests/SwiftDocCTests/Model/SemaToRenderNodeTests.swift

```diff
@@ -3452,4 +3452,23 @@ Document
             ])
         }
     }
+    
+    func testThematicBreak() throws {
+        let source = """
+
+        ---
+
+        """
+        
+        let markup = Document(parsing: source, options: .parseBlockDirectives)
+        let (bundle, context) = try testBundleAndContext(named: "TestBundle")
```

Comment:
Would `testBundleAndContext()` work here? If possible it would be good not build the test bundle only to discard it.